### ISSUE TITLE
Handle relative module path (Parse error on `using ..Foo`)

### ISSUE THREAD

**Issue by Wilfred** [tonyhffong/Lint.jl#71]
I've noticed that when parsing REPL.jl from Julia's source code, Lint.jl gives several errors:

```
UndefVarError(:AnyDict)
$(Expr(:import, :Base, :AnyDict))
ParseError("invalid identifier name \"...\"")
$(Expr(:import, :., :., :LineEdit, :CompletionProvider))
ParseError("invalid identifier name \"...\"")
$(Expr(:import, :., :., :LineEdit, :HistoryProvider))
ParseError("invalid identifier name \"...\"")
$(Expr(:import, :., :., :LineEdit, :add_history))
ParseError("invalid identifier name \"...\"")
$(Expr(:import, :., :., :LineEdit, :complete_line))
ParseError("invalid identifier name \"...\"")
```

Given this is legal Julia code, is the bug in Lint.jl or the parser?

**Comment by TotalVerb** [comment]
Somewhere along the line, this has been fixed:

```julia
lintstr("using ..Foo")
```

no longer returns ParseErrors.

The linter still fails on REPL.jl, but that is https://github.com/JuliaLang/julia/issues/21088.
